code/datums/station_traits/positive_traits.dm

```diff
@@ -0,0 +1,94 @@
+#define PARTY_COOLDOWN_LENGTH_MIN 6 MINUTES
+#define PARTY_COOLDOWN_LENGTH_MAX 12 MINUTES
+
+
+/datum/station_trait/lucky_winner
+	name = "Lucky winner"
+	trait_type = STATION_TRAIT_POSITIVE
+	weight = 1
+	show_in_report = TRUE
+	report_message = "Your station has won the grand prize of the annual station charity event. Free snacks will be delivered to the bar every now and then."
+	trait_processes = TRUE
+	COOLDOWN_DECLARE(party_cooldown)
+
+/datum/station_trait/lucky_winner/on_round_start()
+	. = ..()
+	COOLDOWN_START(src, party_cooldown, rand(PARTY_COOLDOWN_LENGTH_MIN, PARTY_COOLDOWN_LENGTH_MAX))
+
+/datum/station_trait/lucky_winner/process(delta_time)
+	if(!COOLDOWN_FINISHED(src, party_cooldown))
+		return
+
+	COOLDOWN_START(src, party_cooldown, rand(PARTY_COOLDOWN_LENGTH_MIN, PARTY_COOLDOWN_LENGTH_MAX))
+
+	var/area/area_to_spawn_in = pick(GLOB.bar_areas)
+	var/turf/T = pick(area_to_spawn_in.contents)
+
+	var/obj/structure/closet/supplypod/centcompod/toLaunch = new()
+	var/obj/item/pizzabox/pizza_to_spawn = pick(list(/obj/item/pizzabox/margherita, /obj/item/pizzabox/mushroom, /obj/item/pizzabox/meat, /obj/item/pizzabox/vegetable, /obj/item/pizzabox/pineapple))
+	new pizza_to_spawn(toLaunch)
+	for(var/i in 1 to 6)
+		new /obj/item/reagent_containers/food/drinks/beer(toLaunch)
+	new /obj/effect/pod_landingzone(T, toLaunch)
+
+/datum/station_trait/galactic_grant
+	name = "Galactic grant"
+	trait_type = STATION_TRAIT_POSITIVE
+	weight = 5
+	show_in_report = TRUE
+	report_message = "Your station has been selected for a special grant. Some extra funds has been made available to your cargo department."
+
+/datum/station_trait/galactic_grant/on_round_start()
+	var/datum/bank_account/cargo_bank = SSeconomy.get_dep_account(ACCOUNT_CAR)
+	cargo_bank.adjust_money(rand(2000, 5000))
+
+/datum/station_trait/premium_internals_box
+	name = "Premium internals boxes"
+	trait_type = STATION_TRAIT_POSITIVE
+	weight = 10
+	show_in_report = TRUE
+	report_message = "The internals boxes for your crew have been filled with bonus equipment."
+	trait_to_give = STATION_TRAIT_PREMIUM_INTERNALS
+
+/datum/station_trait/bountiful_bounties
+	name = "Bountiful bounties"
+	trait_type = STATION_TRAIT_POSITIVE
+	weight = 5
+	show_in_report = TRUE
+	report_message = "It seems collectors in this system are extra keen to on bounties, and will pay more to see their completion."
+
+/datum/station_trait/bountiful_bounties/on_round_start()
+	SSeconomy.bounty_modifier *= 1.2
+
+/datum/station_trait/strong_supply_lines
+	name = "Strong supply lines"
+	trait_type = STATION_TRAIT_POSITIVE
+	weight = 5
+	show_in_report = TRUE
+	report_message = "Prices are low in this system, BUY BUY BUY!"
+	blacklist = list(/datum/station_trait/distant_supply_lines)
+
+
+/datum/station_trait/strong_supply_lines/on_round_start()
+	SSeconomy.pack_price_modifier *= 0.8
```

Comment:
Ath was a bit flippant about it, but this does seem a bit dangerous. Guess it is reasonably rate tho